static/app/icons/iconBusiness.tsx

```diff
@@ -69,12 +65,6 @@ interface Props extends SVGIconProps {
   withShine?: boolean;
 }
 
-const IconBusiness = forwardRef<SVGSVGElement, Props>((props, ref) => (
```

Comment:
Is there somewhere we're passing in a `ref` that will break now? I don't think types would necessarily catch that.

Reply:
There was only one instance I could find and there was no ref being passed. I'll do another pass to make sure

Reply:
I'm adding it back as it appears most components expose the ref